app/jinja_filters.py

```diff
@@ -43,6 +43,40 @@ def format_date(value):
     return value.strftime('%-d %B %Y')
 
 
+@blueprint.app_template_filter()
+def format_conditional_date(date1=None, date2=None):
+    """
+    This function format_conditional_date accepts two dates, a user submitted date and a metadata date
+
+    :param dates: a list of dates first one is user entered, second is metadata date
+    :return: the value of the date to be piped
+    """
+    dates = ([date1, date2])
+
+    if not len(dates):
+        raise Exception("Empty dates passed to format_conditional_dates filter")
```

Comment:
I don't think this Exception can ever get raised. Both `date1` and `date2` default to `None` if not passed, so dates will always be of length 2?